packages/realm-app-importer/src/AppImportServer.ts

```diff
@@ -79,9 +78,10 @@ export class AppImportServer {
 
   public get url(): string | undefined {
     const address = this.server.address();
-    if (typeof address === "object" && address?.family === "IPv4") {
+    if (typeof address === "object" && address !== null) {
```

Comment:
This was a fix for newer version of node.  Seems it would default to IPv6, which is fine.